presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java

```diff
@@ -34,7 +34,7 @@
 
     public TestingKafka()
     {
-        container = new KafkaContainer("5.2.1");
+        container = new KafkaContainer("5.4.1");
```

Comment:
Is this related to kafka client update?

Reply:
its for version testing.. that its last version of confluent plataform, that have same kafka version of client